src/Illuminate/Database/Query/Builder.php

```diff
@@ -1228,7 +1228,7 @@ public function addWhereExistsQuery(self $query, $boolean = 'and', $not = false)
     {
         $type = $not ? 'NotExists' : 'Exists';
 
-        $this->wheres[] = compact('type', 'operator', 'query', 'boolean');
```

Comment:
why are you removing this?

Reply:
this function doesn't contain variable with name `operator`